.changeset/nice-apricots-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': minor
```

Comment:
@Rugvip is this considered a patch or minor?

Reply:
`patch` since it's a bug fix with no impact on external API